### PR TITLE
Pending deprecate algorithms

### PR DESCRIPTION
**Pull request by manoelmarques**

### CODE REVIEW

File: releasenotes/notes/pending-deprecate-min-eigen-solvers-fa4341e1014e4df0.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+deprecations:
+  - |
+    Module :mod:`qiskit.algorithms.minimum_eigen_solvers` is pending deprecation and
+    is superseded by :mod:`qiskit.algorithms.minimum_eigensolvers`
```
Comment (woodsp-ibm) [inline releasenotes/notes/pending-deprecate-min-eigen-solvers-fa4341e1014e4df0.yaml:5]:
```suggestion
    Module :mod:`qiskit.algorithms.minimum_eigen_solvers` is pending deprecation and
    will be deprecated in a future version and eventually removed. The module and its function  
    are superseded by :mod:`qiskit.algorithms.minimum_eigensolvers`
```

**Comment by woodsp-ibm** [comment]
LGTM like #8651 lets get the new algos merged and merge this afterwards.

**Comment by jakelishman** [comment]
Just a minor comment: if there's no merge conflicts and no test failures, there's no need to "update branch" - the bot will automatically handle it efficiently when the PR is tagged for merge, and in the interim, doing it jams CI for other PRs.

**Review by woodsp-ibm — APPROVED** [review]
The primitive versions replacing these now pending deprecated algos have been merged so this can be merged now too.

**Comment by jakelishman** [comment]
@Mergifyio requeue

**Pull request merged**